### DISCUSSION TITLE
My Doks website not rendering properly  when I deploy it on cloudflare from github

### DISCUSSION THREAD

**Discussion by xT0mR0x** [thuliteio/doks#1023] [General]
on localhost:1313 it's look like how it should be but on cloudflare It's looks like there is no style.. I suspect the node package.
anyone familiar with this?

build log: 

<pre class="c_jr c_bu c_ka c_nz c_nx c_ny c_de c_sb c_sc c_sd c_se c_hq c_hr c_rm c_bw c_sf" style="font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none; box-sizing: border-box; border-radius: 0px; color: rgb(49, 49, 49); display: block; font-size: 12px; margin: 0px; overflow: auto; padding: 16px 32px; width: 704.5px; background-color: rgb(242, 242, 242); border-width: 0px 0px 1px; border-style: solid; border-color: rgb(217, 217, 217); font-family: monaco, courier, monospace; height: 500px; caret-color: rgb(49, 49, 49);">

17:51:09.607 | Cloning repository...
-- | --
17:51:10.798 | From https://github.com/xT0mR0x/doks
17:51:10.798 | * branch            b17d05ce2277afa43212f3789ea0c8cb36a960d8 -> FETCH_HEAD
17:51:10.799 |  
17:51:10.936 | HEAD is now at b17d05c Update .gitignore
17:51:10.936 |  
17:51:11.088 |  
17:51:11.116 | Success: Finished cloning repository files
17:51:11.919 | Installing dependencies
17:51:11.931 | Python version set to 2.7
17:51:15.658 | Downloading and installing node v16.16.0...
17:51:16.131 | Downloading https://nodejs.org/dist/v16.16.0/node-v16.16.0-linux-x64.tar.xz...
17:51:16.604 | Computing checksum with sha256sum
17:51:16.735 | Checksums matched!
17:51:21.246 | Now using node v16.16.0 (npm v8.11.0)
17:51:21.729 | Started restoring cached build plugins
17:51:21.744 | Finished restoring cached build plugins
17:51:22.341 | Attempting ruby version 2.7.1, read from environment
17:51:26.222 | Using ruby version 2.7.1
17:51:26.591 | Using PHP version 5.6
17:51:26.759 | 5.2 is already installed.
17:51:26.787 | Using Swift version 5.2
17:51:26.788 | Started restoring cached node modules
17:51:26.804 | Finished restoring cached node modules
17:51:27.327 | Installing NPM modules using NPM version 8.11.0
17:51:27.745 | npm WARN config tmp This setting is no longer used.  npm stores temporary files in a special
17:51:27.746 | npm WARN config location in the cache, and they are managed by
17:51:27.746 | npm WARN config     [`cacache`](http://npm.im/cacache).
17:51:28.161 | npm WARN config tmp This setting is no longer used.  npm stores temporary files in a special
17:51:28.162 | npm WARN config location in the cache, and they are managed by
17:51:28.162 | npm WARN config     [`cacache`](http://npm.im/cacache).
17:51:39.948 |  
17:51:39.948 | > @hyas/doks@0.5.0 postinstall
17:51:39.948 | > hugo-installer --version otherDependencies.hugo --extended --destination node_modules/.bin/hugo
17:51:39.949 |  
17:51:40.482 |  
17:51:40.482 | Hugo Installer
17:51:40.482 |  
17:51:40.482 | > Checking for existing binary
17:51:40.493 | > Downloading binary from "https://github.com/gohugoio/hugo/releases/download/v0.107.0/hugo_extended_0.107.0_Linux-64bit.tar.gz"
17:51:41.297 | > Downloading checksum from "https://github.com/gohugoio/hugo/releases/download/v0.107.0/hugo_0.107.0_checksums.txt"
17:51:41.659 | > Verifying binary checksum
17:51:41.660 | > Extracting binary to disk
17:51:42.398 | > Verifying binary health
17:51:42.478 |  
17:51:42.478 | Hugo is now available in "node_modules/.bin/hugo".
17:51:42.479 |  
17:51:42.479 | - Version       0.107.0
17:51:42.479 | - Extended      Yes
17:51:42.479 | - OS            linux
17:51:42.479 | - Architecture  x64
17:51:42.479 |  
17:51:42.479 | hugo v0.107.0-2221b5b30a285d01220a26a82305906ad3291880+extended linux/amd64 BuildDate=2022-11-24T13:59:45Z VendorInfo=gohugoio
17:51:42.480 |  
17:51:42.480 | Success!
17:51:42.523 |  
17:51:42.523 | added 583 packages, and audited 584 packages in 14s
17:51:42.523 |  
17:51:42.523 | 90 packages are looking for funding
17:51:42.524 | run `npm fund` for details
17:51:42.526 |  
17:51:42.526 | found 0 vulnerabilities
17:51:42.540 | NPM modules installed
17:51:43.137 | npm WARN config tmp This setting is no longer used.  npm stores temporary files in a special
17:51:43.137 | npm WARN config location in the cache, and they are managed by
17:51:43.137 | npm WARN config     [`cacache`](http://npm.im/cacache).
17:51:43.157 | Installing Hugo 0.54.0
17:51:43.873 | Hugo Static Site Generator v0.54.0-B1A82C61A/extended linux/amd64 BuildDate: 2019-02-01T10:04:38Z
17:51:43.878 | Started restoring cached go cache
17:51:43.899 | Finished restoring cached go cache
17:51:44.051 | go version go1.14.4 linux/amd64
17:51:44.067 | go version go1.14.4 linux/amd64
17:51:44.070 | Installing missing commands
17:51:44.070 | Verify run directory
17:51:44.071 | Executing user command: npm run build
17:51:44.565 | npm WARN config tmp This setting is no longer used.  npm stores temporary files in a special
17:51:44.565 | npm WARN config location in the cache, and they are managed by
17:51:44.565 | npm WARN config     [`cacache`](http://npm.im/cacache).
17:51:44.583 |  
17:51:44.584 | > @hyas/doks@0.5.0 prebuild
17:51:44.584 | > npm run clean
17:51:44.584 |  
17:51:45.003 | npm WARN config tmp This setting is no longer used.  npm stores temporary files in a special
17:51:45.003 | npm WARN config location in the cache, and they are managed by
17:51:45.004 | npm WARN config     [`cacache`](http://npm.im/cacache).
17:51:45.019 |  
17:51:45.020 | > @hyas/doks@0.5.0 clean
17:51:45.020 | > shx rm -rf public resources
17:51:45.021 |  
17:51:45.176 |  
17:51:45.176 | > @hyas/doks@0.5.0 build
17:51:45.176 | > exec-bin node_modules/.bin/hugo/hugo --gc --minify
17:51:45.176 |  
17:51:45.363 | Start building sites …
17:51:45.363 | hugo v0.107.0-2221b5b30a285d01220a26a82305906ad3291880+extended linux/amd64 BuildDate=2022-11-24T13:59:45Z VendorInfo=gohugoio
17:51:48.533 | WARN 2023/03/07 15:51:48 "/opt/buildhome/repo/content/en/docs/my tutorials/Introduction/index.md:1:1": duplicate menu entry with identifier "ipsum-898b778d9e4b73c456e9d6b443abb1f3" in menu "docs"
17:51:48.533 | WARN 2023/03/07 15:51:48 "/opt/buildhome/repo/content/en/docs/my tutorials/how/doks/install.md:1:1": duplicate menu entry with identifier "ipsum-898b778d9e4b73c456e9d6b443abb1f3" in menu "docs"
17:51:50.107 |  
17:51:50.107 | \| EN
17:51:50.108 | -------------------+-----
17:51:50.108 | Pages            \| 40
17:51:50.108 | Paginator pages  \|  0
17:51:50.108 | Non-page files   \|  4
17:51:50.108 | Static files     \| 94
17:51:50.108 | Processed images \| 22
17:51:50.108 | Aliases          \|  5
17:51:50.108 | Sitemaps         \|  1
17:51:50.109 | Cleaned          \|  0
17:51:50.109 |  
17:51:50.109 | Total in 4799 ms
17:51:50.150 | Finished
17:51:50.151 | Found Functions directory at /functions. Uploading.
17:51:50.809 |  
17:51:50.927 | ✘ [ERROR] No routes found when building Functions directory: /opt/buildhome/repo/functions
17:51:50.928 |  
17:51:50.928 |  
17:51:50.928 | If you think this is a bug then please create an issue at https://github.com/cloudflare/workers-sdk/issues/new/choose
17:51:50.942 | Warning: Wrangler did not find routes when building functions. Skipping.
17:51:50.943 | Validating asset output directory
17:51:51.679 | Deploying your site to Cloudflare's global network...
17:51:52.664 | Parsed 0 valid redirect rules.
17:51:52.664 | Parsed 1 valid header rule.
17:51:53.912 | Uploading... (0/167)
17:51:53.913 |  
17:51:53.938 | AAGPOST /pages/assets/upload
17:51:53.938 | Uploading... (0/167)
17:51:53.939 |  
17:51:53.979 | AAGPOST /pages/assets/upload
17:51:53.979 | Uploading... (0/167)
17:51:53.979 |  
17:51:54.006 | AAGPOST /pages/assets/upload
17:51:54.006 | Uploading... (0/167)
17:51:54.007 |  
17:51:56.138 | AAGresult: null
17:51:56.139 | Uploading... (0/167)
17:51:56.139 |  
17:51:56.140 | AAG Uploading... (56/167)
17:51:56.141 |  
17:51:56.833 | AAGresult: null
17:51:56.833 | Uploading... (56/167)
17:51:56.834 |  
17:51:56.834 | AAG Uploading... (112/167)
17:51:56.834 |  
17:51:56.857 | AAGresult: null
17:51:56.858 | Uploading... (112/167)
17:51:56.858 |  
17:51:56.859 | AAG Uploading... (167/167)
17:51:56.859 |  
17:51:56.861 | ✨ Success! Uploaded 167 files (3.47 sec)
17:51:56.861 |  
17:51:57.415 | ✨ Upload complete!
17:51:58.954 | Success: Assets published!
17:51:59.672 | Success: Your site was deployed!

</pre>

**Comment by h-enk** [comment]
Does this help: https://github.com/h-enk/doks/discussions/696#discussioncomment-2611164

**Reply by xT0mR0x** [reply]
sadly it didn't work, I tried on netlify, same problem.. seems like there is no style at all after deployment..

**Reply by xT0mR0x** [reply]
but this worked #https://github.com/h-enk/doks/discussions/963#discussioncomment-4483106 

all I had to do is to change the baseURL to the netlify domain. XD
